client/app/billing/order/billing-order-tracking.routing.js

```diff
@@ -0,0 +1,20 @@
+import controller from './billing-order-tracking.controller';
+import template from './billing-order-tracking.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing.order', {
+    url: '/order/:orderId',
+    params: {
+      ordersFilter: {
+        value: '',
+      },
+    },
+    template,
+    controller,
+    controllerAs: '$ctrl',
```

Comment:
Not a component ?

Reply:
Could you at least answer before resolving ? 😅

Reply:
yes it's not :)